cpp/apps/CMakeLists.txt

```diff
@@ -119,6 +119,46 @@ macro(open3d_add_app SRC_DIR APP_NAME TARGET_NAME)
     endif()
 endmacro()
 
+macro(open3d_add_app_cmd SRC_DIR APP_NAME)
```

Comment:
```
macro(open3d_add_app_common SRC_DIR APP_NAME TARGET_NAME)

endmacro()

target_sources(xxx PRIVATE
    xxx.cpp
    xxx.cpp
)
```

Reply:
It seems that we must add source file in `add_executable`, so I use the original method (`GLOB`) to sum up the source files inside `open3d_add_app_common `